src/commands/publish/preview/mod.rs

```diff
@@ -18,7 +19,63 @@ struct Preview {
     pub id: String,
 }
 
-pub fn preview(
+fn preview_with_auth(
+    project: &Project,
+    user: &GlobalUser,
+    method: Result<HTTPMethod, failure::Error>,
+    body: Option<String>,
+) -> Result<(), failure::Error> {
+    let create_address = format!(
+        "https://api.cloudflare.com/client/v4/accounts/{}/workers/scripts/{}/preview",
+        project.account_id, project.name
+    );
+
+    let client = http::auth_client(user);
+
+    commands::build(&project)?;
+
+    let script_upload_form = publish::build_script_upload_form(project)?;
+
+    log::info!("💩");
```

Comment:
🤣

Reply:
lol i gotta stop asking for review on WIP PRs